dev.sh

```diff
@@ -1,7 +1,14 @@
 #!/bin/sh
 
 # Always try to pull most recent image
-docker push 642631414762.dkr.ecr.eu-west-1.amazonaws.com/frontend-dev:latest
+ecrLogin=$(aws ecr get-login --profile frontend) 
+if [ $? -eq 255  ] 
+then
+    echo "Please use Janus to get credentials"
+    exit 1;
+fi
```

Comment:
Forcing aws credentials to be valid. You need it to run the apps anyway so I don't think it is an overkill to ask for them when running the container.
It would only be an issue if you want to hack around without connection. Let me know if you think this is a problem